core/notifications/src/notification_event/circle_grew.rs

```diff
@@ -1,5 +1,7 @@
 use serde::{Deserialize, Serialize};
 
+use rust_i18n::t;
```

Comment:
We try to group external deps together. This should be just below serde

Reply:
Addressed